utmap-client/src/components/pages/CalendarPage.js

```diff
@@ -295,13 +327,27 @@ function CalendarPage() {
 					closePopup={handleCloseEventInfo}
 					handleEdit={handleEditEventForm}
 					refreshEvents={refreshEvents}
+					errorCallback={errorCallback}
 				/>
 			</Dialog>
 
 			{/* Popup box for grouped event info */}
 			<Dialog open={openGroupedList} onClose={handleCloseGroupedList}>
 				<EventList eventList={groupedEvent}/>
 			</Dialog>
+
+			{/* Error Handling snackbars */}
+			<Snackbar open={openSnackBar500} autoHideDuration={4000} onClose={handleCloseSnackBar500}>
+        		<Alert elevation={6} variant="filled"  onClose={handleCloseSnackBar500} severity="error">
+					500 ERROR: Try again later!
+				</Alert>
+      		</Snackbar>
+
+			<Snackbar open={openSnackBar400} autoHideDuration={4000} onClose={handleCloseSnackBar400}>
+        		<Alert elevation={6} variant="filled"  onClose={handleCloseSnackBar400} severity="error">
+          			400 ERROR: Missing values! Change Form.
```

Comment:
Can you change the error message for 400? It's confusing (I'm not sure what "Change form" means) and it's not accurate for 404 errors. Let's say one user deletes an event and another user tries to update the same event, which will trigger a 404 error. The message should be something relevant to it.